fineract-provider/build.gradle

```diff
@@ -192,6 +194,27 @@ jacocoTestReport{
     }
 }
 
+spotless {
+  format 'misc', {
+   target '**/*.md', '**/*.properties', '**/*.gradle', '**/.gitignore', '**/*.yml', '**/*.xml', '**/**.json', '**/*.sql'
+   targetExclude '**/build/**', '**/bin/**'
+   indentWithSpaces(4)
+   trimTrailingWhitespace()
```

Comment:
could `trimTrailingWhitespace()` + `endWithNewline()` be moved "out" of 'misc' and 'java' at the start (or end) of the spotless configuration, together with `lineEndings`, so that it applies to everything? Or is that not how this configuration works?

Reply:
I already tried doing that, it won't let us do it :(

Reply:
OK, resolving.